src/Confluent.Kafka/Serialization/IgnoreDeserializer.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.Serialization
+{
+    /// <summary>
+    ///     A 'deserializer' that returns null regardless of the input data.
```

Comment:
Why the fnutts? (quotes)

Reply:
because it doesn't deserialize anything.

Reply:
I like having the quotes because they are informational - there is a good reason they are there and it makes people wonder why.